src/js/shared/config_file.js

```diff
@@ -30,6 +30,21 @@ const SearchProv = {
 };
 
 const ConfigFile = {
+  decrypt: function (data, key) {
+    // Convert string key to bytes and make it multiple of 16 bytes.
+    key = aesjs.padding.pkcs7.pad(aesjs.utils.utf8.toBytes(key));
```

Comment:
> All keys must be 128 bits (16 bytes), 192 bits (24 bytes) or 256 bits (32 bytes) long.
> from: https://www.npmjs.com/package/aes-js#keys

Shouldn't your `key` be 128, 192, or 256 bits long? I suspect this code is padding with Zeroes which probably not what we want here. 

You should also specify an IV. See the docs here: https://www.npmjs.com/package/aes-js#cbc---cipher-block-chaining-recommended